CMSgov/easi-app
# Easi 1368/system profile components

# EASI-1368



## Changes proposed in this pull request

- Added a "System Profile Health Card" component.
  - NOTE: This component is not complete, and this should be considered a prototype.



## Reviewer Notes



## Setup & How to test

- Pull the branch locally and test it
  - Visit the `/sandbox` route after logging in to view the components.
- Check that all code is adequately covered by tests
- Make comments, even if it's minor!



## PR Author Review Checklist

functions, or e2e tests as necessary
the [rotor menu](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)
- For any new migrations/schema changes:

## Screenshots

![image](https://user-images.githubusercontent.com/15203744/144070532-b7ccee80-a510-44f5-b28d-c7ade4ca4976.png)
